operations/node_management/Swap_a_Compute_Blade_with_a_Different_System_Using_SAT.md

```diff
@@ -0,0 +1,358 @@
+   Review *HPE Cray EX Coolant Service Procedures H-6199*. If using the hand pump, review procedures in the *HPE Cray EX Hand Pump User Guide H-6200* ([HPE Support](https://internal.support.hpe.com/)).
+1. Install the blade from the source system in a storage rack or leave it on the cart.
+
+## Prepare the destination system blade for removal
+
+1. Using the work load manager (WLM), drain running jobs from the affected nodes on the blade. Refer to the vendor documentation for the WLM for more information.
+
+1. Use the `sat bootsys` command (in this example, `x1005c0s3`) to shut down the nodes on the target blade. Specify the appropriate component name (xname) and BOS
+   template for the node type in the following command.
+
+   ```bash
+   ncn# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
+   ncn# sat bootsys shutdown --stage bos-operations --bos-limit x1005c0s3 --recursive --bos-templates $BOS_TEMPLATE
+   ```
+
+### Destination: Use SAT to remove the blade from hardware management
+
+1. Use the `sat swap` command to power off the slot and delete the blade's ethernet interfaces and Redfish endpoints from HSM.
+
+   ```bash
+   ncn# sat swap blade -a disable x1005c0s3
+   ```
+
+   The mapping file should be copied to the NCN on the destination system used for the swap procedure, if necessary. In this example, the
+   filename is changed to `ethernet-interface-mappings-src.json` on the destination system for clarity.
+
+   ```bash
+   ncn# scp ethernet-interface-mappings-x1005c0s3-2022-01-01.json <src>-ncn-m001:ethernet-interface-mappings-dest.json
+   ```
+
+### Destination: Swap the blade hardware
+
+1. Remove the blade from destination system install it in a storage cart.
+
+1. Install the blade from the source system into the destination system.
+
+## <a name="bring-up-the-blade-in-the-destination-system"></a>Bring up the blade in the destination system
+
+### Destination: Use SAT to add the blade to hardware management
+
+1. Use the `sat swap` command to map the nodes' ethernet interface MAC addresses to the appropriate IP addresses and component xnames, and begin discovery for the blade.
+
+   The `--src-mapping` and `--dst-mapping` arguments should be used to pass in the ethernet interface mapping files created during the previous steps.
+
+   ```bash
+   ncn# sat swap blade -a enable --src-mapping ethernet-interface-mappings-src.json --dst-mapping ethernet-interface-mappings-x1005c0s3-2022-01-01.json x10005c0s3
+   ```
+
+### Power on and boot the nodes
+
+1. Use `sat bootsys` to power on and boot the nodes. Specify the appropriate BOS template for the node type.
+
+   ```bash
+   ncn# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
+   ncn# sat bootsys boot --stage bos-operations --bos-limit x1005c0s3 --recursive --bos-templates $BOS_TEMPLATE
+   ```
+
+### Check firmware
+
+1. Verify that the correct firmware versions for node BIOS, node controller (nC), NIC mezzanine card (NMC), GPUs, and so on.
```

Comment:
Provide a link in this step to the FAS documentation on how to check these versions.